tests/other/test_export_global_address.c

```diff
@@ -5,8 +5,8 @@
 EMSCRIPTEN_KEEPALIVE int g_foo = 4;
 
 EM_JS(int*, get_foo_from_js, (void), {
-  assert(Module['_g_foo'] !== undefined, "g_foo not exported to JS");
-  return Module['_g_foo'];
+  assert(_g_foo !== undefined, "g_foo not exported to JS");
```

Comment:
maybe assert `> 0`? that would also check it's not 0.

Reply:
This test already verifies that the returned address below.